### DISCUSSION TITLE
Working on sqlc full-time

### DISCUSSION THREAD

**Discussion by kyleconroy** [sqlc-dev/sqlc#2411] [Announcements]
> Cross-posted from the blog: https://sqlc.dev/posts/2023/07/06/working-on-sqlc-full-time/

After developing sqlc in our spare time for the last three years, we're excited to share that we've incorporated our company, Riza[^1], and we'll be working on sqlc full-time!

We've got a bunch of great features planned, some of which shipped in [v1.19.0][2]. Notably we introduced the [`sqlc vet` subcommand][9] which checks your queries against lint expressions to catch common mistakes in otherwise syntactically-valid SQL.

The introduction of `sqlc vet` also marks the first time that sqlc will use a [running database server connection][8] to enhance its functionality. We'll expand on this in future releases, but for now database connections power the built-in `sqlc/db-prepare` vet rule.

We're really excited to get your feedback on this, so if you have thoughts about how to make `sqlc vet` more useful please [let us know][4]!

We also created a new GitHub organization ([sqlc-dev][6]) to house existing and future work. We've already migrated most sqlc-related repositories, but we're waiting to move the [kyleconroy/sqlc][7] repo. The plan is to migrate it in the next week or two. We hope this doesn't cause any disruption but some discomfort during the transition is possible. If you've moved or renamed a Go project on GitHub before, we would definitely appreciate your tips on how to make this change smoothly.

We'll attempt to set a monthly cadence for new sqlc releases from now on. If you'd like to keep up with new releases and other future sqlc developments you can [subscribe to our newsletter][3]. 

Riza is a for-profit corporation, so we’ll be exploring monetization strategies to sustain sqlc development. That being said, we have no intention of changing the license of sqlc. We expect to build auxiliary services that enhance sqlc's usefulness when you opt-in, and hopefully we'll do a good-enough job to earn your trust as customers.

We're very open to feedback about this or anything else, so please send us a message via email to [hello@sqlc.dev][4] or in the [sqlc Discord server][5] if you'd like to chat.

Sincerely,
Kyle and Andrew

[^1]: The name doesn't mean anything in English but does mean ["root" in Greek][1]. We needed a name that wouldn't tie us too closely to "SQL" in case we work on other non-SQL tools in the future. It's a little confusing, we know. And honestly it's almost not worth mentioning, but we don't want you to be alarmed when you see "Riza, Inc." floating around in copyright notices.

[1]: https://en.wiktionary.org/wiki/%CF%81%CE%AF%CE%B6%CE%B1
[2]: https://sqlc.dev/posts/2023/07/06/sqlc-one-point-nineteen-released
[3]: https://sqlc.dev/newsletter/
[4]: mailto:hello@sqlc.dev
[5]: https://discord.gg/EcXzGe5SEs
[6]: https://github.com/sqlc-dev
[7]: https://github.com/kyleconroy/sqlc
[8]: https://docs.sqlc.dev/en/v1.19.0/reference/config.html#database
[9]: https://docs.sqlc.dev/en/stable/howto/vet

**Comment by DarkHeros09** [comment]
great news 🎉🎉

**Comment by omissis** [comment]
Congrats! 🎉

**Comment by go-mez** [comment]
Keep up the good work!

**Comment by jwc-clinnection** [comment]
Awesome news!
